### PR TITLE
[ci] automatically update PRs with format fixes

### PR DESCRIPTION
**Pull request by titusfortner**
### Status
Well, now I'm trying to see if I can do something complicated and add a label to rerun tests if there has been an automatic reformatting. It's set up so that it shouldn't end up in a loop. Not sure it is worth going this far, though.

### Description
Uses what was done for automatically pinning browsers
Creates a patch file with format changes
In a new job applies the patch file and commits it
pushes the resulting commit back to the PR

### Motivation and Context
It's a pain to ask people to fix formatting in their PRs

I introduced formatting bugs in the PR so this should push to the PR. 🤞 

There we go, it works!
https://github.com/SeleniumHQ/selenium/pull/13514/commits/36ccb886d4d4a99972f8159193fed0ebf2d6e04c

### CODE REVIEW

**Comment by titusfortner** [comment]
well, this probably works. We'll see once selenium-ci gets off its butt and signs the CLA.

**Comment by titusfortner** [comment]
Hmm, but I'm expecting this to re-run the tests... I wonder why it does not...

**Comment by p0deje** [comment]
> I introduced formatting bugs in the PR so this should push to the PR. 🤞


This would only work on Selenium PRs, not from forks. Is this fine for you?

**Comment by titusfortner** [comment]
@p0deje why wouldn't it work for forks?

**Comment by p0deje** [comment]
> @p0deje why wouldn't it work for forks?

@titusfortner Because Selenium bot would not have permissions to push to the forked repository, would it?

**Comment by titusfortner** [comment]
There's a checkbox for
"Allow edits from maintainers" that I believe is checked by default. But I should test something that has it turned off.

**Comment by titusfortner** [comment]
Hmm, the label to rerun the tests didn't kick off new tests automatically. Does this need to be on trunk first? Maybe this shouldn't be automatic, but run manually by adding the label? I'm going to test this more on my fork...

File: .github/workflows/ci-rbe.yml [hunk 1]
```diff
@@ -6,21 +6,95 @@ on:
     branches:
       - trunk
   workflow_dispatch:
+  issues:
+    types: [ labeled ]
```
Comment (diemol) [inline .github/workflows/ci-rbe.yml:10]: Will this run when any label is applied?

**Comment by titusfortner** [comment]
I spent a *lot of time exploring options here, and I'm not happy with anything.

To prevent infinite loops, github won't allow tests to rerun from the same workfkow. I can get it to label and rerun, but the tests don't get attached to the commit that was just made in the UI. Either way, the PR shows zero checks in the checks tab (what is in this PR)

I'm out of ideas and tired of working on this, so I'm giving up. 😄

**Pull request closed without merging**